tasks/electron-react-ts/public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self'" />
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-eval'; object-src 'self'" />
```

Comment:
Not sure if it's worth adding a small bit about this change in the README, or maybe just pointing people to https://content-security-policy.com/